src/cmd/ksh93/sh/fault.c

```diff
@@ -387,7 +387,6 @@ void sh_chktrap(Shell_t *shp) {
         }
     }
     if (!shp->sigflag) return;
-    if (shp->sigflag[SIGALRM] & SH_SIGALRM) sh_timetraps(shp);
```

Comment:
Both `SH_SIGALRM` and `SH_SIGTSTP` are defined as `0200`. I suspect there are going to be side effects of this change.